File: B.cpp/Product.cpp
#include<string>
#include<iostream>


using namespace std;

class Product{
    private :
        int price;
        int id_product;

    public :
        Product(){

        }

        void setPrice(int price){
            this->price = price;
        }

        int getPrice(){
            return this->price;
        }

        void setId_Product(int id_product){
            this->id_product = id_product;
        }

        int getId_Product(){
            return this->id_product;
        }

        void output_product(){
            cout << "Id Product                 : " << this->getId_Product() << endl;
            cout << "Price                      : " << this->getPrice() << endl;
        }

        ~Product(){}


        
};

File: Poin B.py/Product.py
#multi level inheritence
class Product :
    #inisialisasi atribut
    __Id_Product = 0
    __Price = 0
    
    #constructor
    def __init__(self, id_product = 0, price = 0):
        self.__Id_Product = id_product
        self.__Price = price
    
    
    #get set methods
    def setId_Product(self, id_product = 0):
        self.__Id_Product = id_product
    
    def getId_Product(self) :
        return self.__Id_Product
    
    def setPrice(self, price = 0) :
        self.__Price = price
        
    def getPrice(self) :
        return self.__Price 
    
    #output method
    def outputProduct(self) :
        print("Id Product                    : "+ str(self.getId_Product()))
        print("Price                         : "+ str(self.getPrice()))
    

        
    
        